examples/rcn.py

```diff
@@ -0,0 +1,452 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # RCN implementation using the PGMax package
+
+# %% [markdown]
+# RCN is a neuroscience based, data-efficient, interpretable object detection/vision system. It is a probabilistic graphical model (PGM) that trains in minutes and generalizes well (for certain traits). It can reason about, and is robust to, occlusion. It can also imagine and interpret scenes.
+#
+# Please refer to the [science](https://www.science.org/doi/10.1126/science.aag2612) paper for more details. In this notebook, we implement a two-level RCN model on a small subset of the MNIST dataset.
+
+# %%
+# %matplotlib inline
+import os
+import time
+from typing import Tuple
+
+import jax
+import matplotlib.pyplot as plt
+import numpy as np
+from jax import numpy as jnp
+from jax import tree_util
+from joblib import Memory
+
+memory = Memory("./example_data/tmp")
```

Comment:
Does this have to be here? If not maybe move it after all the imports

Reply:
Seems like we are ignoring E402 in flake8 for some reason....Better to move it after all imports

Reply:
Moved to after imports.